### ISSUE TITLE
Details for hyperparameter for pre-training.

### ISSUE THREAD

**Issue by JonghwanMun** [X-PLUG/mPLUG-Owl#140]
In Table 4 of the appendix, detailed hyper-parameters for pre-training is given.
![image](https://github.com/X-PLUG/mPLUG-Owl/assets/6416693/7cf39049-d634-4ca7-bd7f-ca2f52ebeec3)

However, in Section 4 of the paper, for pre-training, it is mentioned that 0.0001 (1e-4) learning rate, 0.1 weight decay and 2k warm-up steps are used.

Which values are correct?

**Comment by MAGAer13** [comment]
The section 4. We will correct it.
